packages/aws-cdk/lib/api/deploy-stack.ts

```diff
@@ -73,7 +75,8 @@ export async function deployStack(options: DeployStackOptions): Promise<DeploySt
     TemplateURL: bodyParameter.TemplateURL,
     Parameters: params,
     RoleARN: options.roleArn,
-    Capabilities: [ 'CAPABILITY_IAM', 'CAPABILITY_NAMED_IAM', 'CAPABILITY_AUTO_EXPAND' ]
+    Capabilities: [ 'CAPABILITY_IAM', 'CAPABILITY_NAMED_IAM', 'CAPABILITY_AUTO_EXPAND' ],
+    Tags: options.tags
```

Comment:
What happens if this gets `undefined`?

Reply:
Probably one for integration. I have manually tested and means no tags. Meaning, any previous tags applied will be removed (what I think makes sense)

If no tags are provided by either way, via aspects or arguments, the cloudformation template doesn't populate the "Tags" (at least from the synth and the one in the aws cloudformation template output).